packages/p4pp/p4pp.0.1.11/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "P4PP: Preprocessor for P4 Language"
+maintainer: "jnfoster@cs.cornell.edu"
+authors: ["Nate Foster <jnfoster@cs.cornell.edu>"]
+available: opam-version >= "2.1.0"
+homepage: "https://github.com/cornell-netlab/p4pp"
+dev-repo: "git+https://github.com/cornell-netlab/p4pp/"
+bug-reports: "https://github.com/cornell-netlab/p4pp/issues"
+license: "Apache-2.0"
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+]
+depends: [
+  "ocaml" {>= "4.09.1"}
+  "dune" {>= "1.4"}
+  "menhir" {>= "20180523" & <= "20211230"}
+  "core" {>= "v0.13.0" & < "v0.15.0"}
```

Comment:
These constraints seem rather strict. Does this package really only works with outdated versions of those libraries?
```suggestion
  "menhir" {>= "20180523"}
  "core" {>= "v0.13.0"}
```

Reply:
Ah, yeah, this was a weird way to fix a downstream problem. I think I can resolve it without a package update.